packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -125,24 +162,25 @@ export class EthereumAnchorService implements AnchorService {
    * @param params - a RequestAnchorParams object
    * @private
    */
-  private _makeAnchorRequest(params: RequestAnchorParams): Observable<AnchorServiceResponse> {
+  private _makeAnchorRequest(
+    streamID: StreamID,
+    tip: CID,
+    carFile: CAR
+  ): Observable<AnchorServiceResponse> {
     return defer(() =>
       from(
         this.sendRequest(this.requestsApiEndpoint, {
           method: 'POST',
-          body: {
-            streamId: params.streamID.toString(),
-            cid: params.tip.toString(),
-            timestamp: params.timestampISO,
-          },
+          headers: { 'Content-Type': 'application/vnd.ipld.car' },
+          body: carFile.bytes,
```

Comment:
Fwiw, I'm not sure if fetch works with Uint8Arrays. I've never actually used it to send binary data. 🤞